robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -343,25 +400,35 @@ def get_closest_possible_person_entity(self, center_point=None, radius=float('in
     #                                  KINECT INTEGRATION AND PERCEPTION
     # ----------------------------------------------------------------------------------------------------
 
-    def update_kinect(self, area_description="", background_padding=0):
+    def update_kinect(self, area_description="", background_padding=0, fit_supporting_entity=True):
         """
         Update ED based on kinect (depth) images
 
         :param area_description: An entity id or area description, e.g. "a08d537e-e051-11e5-a34e-6cc217ec9f41" or "on_top_of cabinet-11"
         :param background_padding: The maximum distance to which kinect data points are associated to existing objects (in meters).
                Or, in other words: the padding that is added to existing objects before they are removed from the point cloud
+        :param fit_supporting_entity: Fit or not fit the supporting entity
         :return: Update result
         """
         # Check the area description
         if area_description == "":
             rospy.logwarn("No area_description provided for 'update_kinect'. This is probably a bad idea.")
 
         # Save the image (logging)
-        self.save_image(path_suffix=area_description.replace(" ", "_"))
+        # self.save_image(path_suffix=area_description.replace(" ", "_"))
```

Comment:
This was disabled because of RGBD issues. We could maybe re-enable this in combination with https://github.com/tue-robotics/tue_robocup/commit/72e50fdf6b5bafbbf325375b4e57b0d715023c88